src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -287,6 +287,8 @@ public function delete_item_permissions_check( $request ) {
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function delete_item( $request ) {
+		global $wp_registered_widget_updates;
```

Comment:
We need to add this global to the docblock.

Reply:
Done 👍